app/models/local_authority.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class LocalAuthority < ApplicationRecord
+  has_many :schools
```

Comment:
The same as in other pull request - if we delete local authority  - do we expect schools to be deleted as well or do we want them as orphaned records.

Reply:
Going to mark it as resolved - after thinking about it dealing with orphaned school might be easier than dealing with deleted school due to user error